sdks/python/apache_beam/io/gcp/tests/bigquery_matcher.py

```diff
@@ -0,0 +1,110 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Bigquery data verifier for end-to-end test."""
+
+import logging
+
+from hamcrest.core.base_matcher import BaseMatcher
+
+from apache_beam.tests.test_utils import compute_hash
+from apache_beam.utils import retry
+
+# Protect against environments where bigquery library is not available.
+# pylint: disable=wrong-import-order, wrong-import-position
+try:
+  from google.cloud import bigquery
+  from google.cloud.exceptions import GoogleCloudError
+except ImportError:
+  bigquery = None
+# pylint: enable=wrong-import-order, wrong-import-position
+
+MAX_RETRIES = 4
+
+
+def retry_on_http_and_value_error(exception):
+  """Filter allowing retries on Bigquery errors and value error."""
+  if isinstance(exception, GoogleCloudError) or \
```

Comment:
`return isinstance(exception, GoogleCloudError) or isinstance(exception, ValueError)`

You don't need the true false